src/python/nightlyBench.py

```diff
@@ -2003,6 +2034,21 @@ def writeStoredFieldsBenchmarkHTML(storedFieldsBenchmarkData):
     writeKnownChanges(w, pctOffset=250)
     footer(w)
 
+def writeNADFacetBenchmarkHTML(nadFacetBenchmarkData):
+  with open('%s/nad_facet_benchmarks.html' % constants.NIGHTLY_REPORTS_DIR, 'w') as f:
+    w = f.write
+    header(w, 'National Address Database high cardinality faceting benchmarks')
```

Comment:
Maybe hyperlink out to the NAD source website?

Reply:
Will add in next revision